tests/integration_tests/test_chainermn.py

```diff
@@ -395,21 +395,22 @@ def test_suggest_discrete_uniform(storage_mode, comm):
 
     @staticmethod
     @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
-    def test_suggest_int(storage_mode, comm):
-        # type: (str, CommunicatorBase) -> None
+    @pytest.mark.parametrize("enable_log", [False, True])
+    def test_suggest_int(storage_mode, comm, enable_log):
+        # type: (str, CommunicatorBase, bool) -> None
```

Comment:
Please use the Python3-style type hinting.
```suggestion
    def test_suggest_int(storage_mode: str, comm: CommunicatorBase, enable_log: bool) -> None:
```

Reply:
I have updated it. Thanks!